app/src/main/kotlin/co/netguru/android/carrecognition/feature/camera/CameraPresenter.kt

```diff
@@ -23,9 +27,11 @@ import javax.inject.Inject
 import kotlin.math.sqrt
 
 @ActivityScope
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Too many functions inside a/an file/class/object/interface always indicate a violation of the single responsibility principle. Maybe the file/class/object/interface wants to manage to many things at once. Extract functionality which clearly belongs together. SIZE 14 is greater than the threshold 10. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.devguru.co/coding_rules#rule_key=detekt-kotlin%3ATooManyFunctions)